$NetBSD: patch-libelftc_Makefile,v 1.2 2018/10/01 12:36:35 ryoon Exp $

Add a minor version to libelftc.so; otherwise a bootstrapped build does
"ln -sf libelftc.so.1 libelftc.so.1" and then install fails with ELOOP.

This is a bug in bootstrap-mk-files but I'm going to fix it in both
places to avoid demanding rebootstrapping.

--- libelftc/Makefile.orig	2018-09-26 00:03:01.469445303 +0000
+++ libelftc/Makefile
@@ -23,6 +23,7 @@ INCSDIR=	/usr/include
 RELEASE=	0.7.1		# Change this on release branches.
 
 SHLIB_MAJOR=	1
+SHLIB_MINOR=	0
 
 WARNS?=	6
 
